src/helpers/bestseller.js

```diff
@@ -0,0 +1,34 @@
+import {bestSellingTv, inventory} from "../constants/inventory.js";
+
+function screenSizeFormat(){
+    let sizeString = ""
+    for (const size in bestSellingTv.availableSizes) {
+        sizeString += bestSellingTv.availableSizes[size]
+        sizeString += " inch"
+        sizeString += " (" + (bestSellingTv.availableSizes[size]*2.54) + " cm) "
+        sizeString += " | "
+    }
+    return sizeString.split("| ")
+}
+
+function tvName(object){
+    let brand = bestSellingTv.brand
+    let type = bestSellingTv.type
+    let name = bestSellingTv.name
+
+    return brand+" "+type+" - "+name
```

Comment:
Wat ik zelf altijd makkelijker lezen vind:
```suggestion
    return `${brand} ${type} - ${name}`;
```